zebrad/tests/common/lightwalletd/wallet_grpc_test.rs

```diff
@@ -0,0 +1,249 @@
+//! Test all grpc calls a wallet connected to a lightwalletd instance backed by zebra can do.
+//!
+//! This test requires a cached chain state that is partially synchronized, i.e., it should be a
+//! few blocks below the network chain tip height. It also requires a lightwalletd data dir in sync
+//! with the cached chain state.
+//!
+//! Current coverage of all available rpc methods according to `CompactTxStreamer`:
+//!
+//! - `GetLatestBlock`: Covered.
+//! - `GetBlock`: Covered.
+//! - `GetBlockRange`: Covered.
+//!
+//! - `GetTransaction`: Covered.
+//! - `SendTransaction`: Not covered and it will never will, it has its own test.
+//!
+//! - `GetTaddressTxids`: Not covered, need #4216 to be fixed first.
+//! - `GetTaddressBalance`: Covered.
+//! - `GetTaddressBalanceStream`: Not covered.
+//!
+//! - `GetMempoolTx`: Not covered.
+//! - `GetMempoolStream`: Not covered.
+//!
+//! - `GetTreeState`: Not covered, Need #3990
+//!
+//! - `GetAddressUtxos` -= Covered.
+//! - `GetAddressUtxosStream`: Not covered.
```

Comment:
Do we need to open a ticket for the missing gRPC tests?

Reply:
Opened https://github.com/ZcashFoundation/zebra/issues/4278 for this.